apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/MetricsSummary/BigNumberPanels/MeanValuePanel/index.tsx

```diff
@@ -0,0 +1,53 @@
+'use client'
+
+import { Evaluation, EvaluationMeanValue } from '@latitude-data/core/browser'
+import { RangeBadge } from '@latitude-data/web-ui'
+import useEvaluationResultsMeanValue from '$/stores/evaluationResultCharts/evaluationResultsMeanValue'
+
+import Panel from '../Panel'
+
+export default function MeanValuePanel({
+  mean,
+  commitUuid,
+  documentUuid,
+  evaluation,
+}: {
+  commitUuid: string
+  documentUuid: string
+  evaluation: Evaluation
+  mean: EvaluationMeanValue
+}) {
+  const { data } = useEvaluationResultsMeanValue(
+    {
+      commitUuid,
+      documentUuid,
+      evaluationId: evaluation.id,
+    },
+    {
+      revalidateOnFocus: true,
+      revalidateOnMount: false,
+      revalidateOnReconnect: false,
+      refreshWhenOffline: false,
+      refreshWhenHidden: false,
+      refreshInterval: 0,
```

Comment:
I wouldn't add all these protections and in fact i would refetch every x seconds. This computation might be expensive but we have few users at the moment so it's better to optimize for the best experience until we implement.

Reply:
btw refetchinterval won't refetch if none of the store parameters have changed so you actually have to do it like i did [here](https://github.com/latitude-dev/latitude-llm/pull/230/files#diff-05fe786a734cd364c02cc72cf5d7a831e28a60507cdaa0983904fd5eaab798fbL43)

Reply:
A polling

Reply:
> I'm going to work on this after merging.

yes but in the meantime we don't want to break the realtime feeling of the metrics please add the polling

Reply:
done